Python/Canny edge detector/canny edge.py

```diff
@@ -0,0 +1,20 @@
+import cv2
+import numpy as np
+from matplotlib import pyplot as plt
+import tkinter as tk 
+from tkinter import filedialog 
+root = tk.Tk() 

```

Comment:
We don't actually use Tkinter for Scripting? Can you make it a plain script that can be launched from CLI without any GUI?